pkg/process/checks/checks.go

```diff
@@ -35,6 +36,16 @@ type CheckWithRealTime interface {
 	RunWithOptions(cfg *config.AgentConfig, nextGroupID func() int32, options RunOptions) (*RunResult, error)
 }
 
+// Enabled returns true if there is at least one check enabled. This is used to tell if the process agent should be run.
+func Enabled() bool {
+	for _, chk := range All {
+		if chk.Enabled() {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Potentially even better approach would be to just check `len` of the global slice `EnabledChecks`, not a fan of adding a boolean to every check.